config/webhook/kustomization.yaml

```diff
@@ -1,6 +1,6 @@
 resources:
 - manifests.yaml
-- service.yaml
+#- service.yaml
```

Comment:
why is this commented out?

Reply:
It's not used, so I commented it out.  OLM generates the service at deploy-time automagically